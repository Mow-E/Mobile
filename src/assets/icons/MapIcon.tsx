import * as React from 'react';
import Svg, {SvgProps, Line, Circle, Path} from 'react-native-svg';
import IconProps from './IconProps';
import colors from '../../styles/colors';

function SvgComponent(props: SvgProps): JSX.Element {
  return (
    <Svg
      width={24}
      height={24}
      viewBox="0 0 69.470993 67.297112"
      fill="none"
      {...props}>
      <Line
        y1={26.296127}
        x2={0.85401821}
        y2={67.291687}
        stroke={props.stroke}
        x1={1.3100156}
      />
      <Line
        y1={17.829628}
        x2={68.51503}
        y2={58.825188}
        stroke={props.stroke}
        x1={68.971024}
      />
      <Line
        y1={26.296127}
        x2={45.963028}
        y2={67.291687}
        stroke={props.stroke}
        x1={46.419025}
      />
      <Line
        y1={17.829628}
        x2={23.410929}
        y2={58.825188}
        stroke={props.stroke}
        x1={23.866926}
      />
      <Line
        y1={25.82403}
        x2={23.182575}
        y2={17.357529}
        stroke={props.stroke}
        strokeWidth={0.999984}
        x1={0.63054752}
      />
      <Line
        y1={17.356358}
        x2={46.090553}
        y2={25.822845}
        stroke={props.stroke}
        strokeWidth={0.999984}
        x1={23.087475}
      />
      <Line
        y1={26.273958}
        x2={68.869011}
        y2={17.80747}
        stroke={props.stroke}
        strokeWidth={0.999984}
        x1={45.865936}
      />
      <Line
        y1={66.819733}
        x2={22.731525}
        y2={58.35323}
        stroke={props.stroke}
        strokeWidth={0.999984}
        x1={0.1794975}
      />
      <Line
        y1={58.352058}
        x2={46.090553}
        y2={66.818542}
        stroke={props.stroke}
        strokeWidth={0.999984}
        x1={23.087475}
      />
      <Line
        y1={66.819733}
        x2={68.286682}
        y2={58.35323}
        stroke={props.stroke}
        strokeWidth={0.999984}
        x1={45.734657}
      />
      <Circle
        cx={34.412552}
        cy={14.1218}
        r={14.1218}
        fill={props.fill}
        stroke={props.fill}
      />
      <Path
        d="m 34.116455,42.4782 -13.6445,-24.5226 27.7313,0.2453 z"
        fill={props.fill}
        stroke={props.fill}
      />
    </Svg>
  );
}

/**
 * An icon that shows a map.
 * Used as the main icon for the map section(s).
 */
function MapIcon({
  size = 24,
  colored = false,
  darkModeInverted = false,
}: IconProps): JSX.Element {
  const strokeColor = darkModeInverted
    ? colors.gray['300']
    : colors.gray['950'];
  const coloredFillColor = darkModeInverted
    ? colors.red.light
    : colors.red.dark;
  const grayscaleFillColor = darkModeInverted
    ? colors.gray['50']
    : colors.gray['300'];
  const fillColor = colored ? coloredFillColor : grayscaleFillColor;

  return (
    <SvgComponent
      height={size}
      width={size}
      stroke={strokeColor}
      fill={fillColor}
    />
  );
}

export default MapIcon;
