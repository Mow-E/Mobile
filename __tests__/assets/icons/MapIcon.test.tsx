import 'react-native';
import React from 'react';
import MapIcon from '../../../src/assets/icons/MapIcon';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders default correctly', () => {
  const tree = renderer.create(<MapIcon />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders colored correctly', () => {
  const tree = renderer.create(<MapIcon colored />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders uncolored correctly', () => {
  const tree = renderer.create(<MapIcon colored={false} />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders in dark mode correctly', () => {
  const tree = renderer.create(<MapIcon darkModeInverted />).toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders in different sizes', () => {
  const treeSmall = renderer.create(<MapIcon size={24} />).toJSON();
  const treeLarge = renderer.create(<MapIcon size={48} />).toJSON();

  expect(treeSmall).not.toEqual(treeLarge);
});
