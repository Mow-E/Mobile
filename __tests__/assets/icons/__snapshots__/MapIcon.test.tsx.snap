// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders colored correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="#ad2727"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  stroke="#000000"
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={67.297112}
  vbWidth={69.470993}
  width={24}
>
  <RNSVGGroup
    fill={
      {
        "payload": 4289537831,
        "type": 0,
      }
    }
    propList={
      [
        "fill",
        "stroke",
      ]
    }
    stroke={
      {
        "payload": 4278190080,
        "type": 0,
      }
    }
  >
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="1.3100156"
      x2="0.85401821"
      y1="26.296127"
      y2="67.291687"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="68.971024"
      x2="68.51503"
      y1="17.829628"
      y2="58.825188"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="46.419025"
      x2="45.963028"
      y1="26.296127"
      y2="67.291687"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="23.866926"
      x2="23.410929"
      y1="17.829628"
      y2="58.825188"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="0.63054752"
      x2="23.182575"
      y1="25.82403"
      y2="17.357529"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="23.087475"
      x2="46.090553"
      y1="17.356358"
      y2="25.822845"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="45.865936"
      x2="68.869011"
      y1="26.273958"
      y2="17.80747"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="0.1794975"
      x2="22.731525"
      y1="66.819733"
      y2="58.35323"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="23.087475"
      x2="46.090553"
      y1="58.352058"
      y2="66.818542"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="45.734657"
      x2="68.286682"
      y1="66.819733"
      y2="58.35323"
    />
    <RNSVGCircle
      cx="34.412552"
      cy="14.1218"
      fill={
        {
          "payload": 4289537831,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      r="14.1218"
      stroke={
        {
          "payload": 4289537831,
          "type": 0,
        }
      }
    />
    <RNSVGPath
      d="m 34.116455,42.4782 -13.6445,-24.5226 27.7313,0.2453 z"
      fill={
        {
          "payload": 4289537831,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4289537831,
          "type": 0,
        }
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`renders default correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="#d9d9d9"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  stroke="#000000"
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={67.297112}
  vbWidth={69.470993}
  width={24}
>
  <RNSVGGroup
    fill={
      {
        "payload": 4292467161,
        "type": 0,
      }
    }
    propList={
      [
        "fill",
        "stroke",
      ]
    }
    stroke={
      {
        "payload": 4278190080,
        "type": 0,
      }
    }
  >
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="1.3100156"
      x2="0.85401821"
      y1="26.296127"
      y2="67.291687"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="68.971024"
      x2="68.51503"
      y1="17.829628"
      y2="58.825188"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="46.419025"
      x2="45.963028"
      y1="26.296127"
      y2="67.291687"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="23.866926"
      x2="23.410929"
      y1="17.829628"
      y2="58.825188"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="0.63054752"
      x2="23.182575"
      y1="25.82403"
      y2="17.357529"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="23.087475"
      x2="46.090553"
      y1="17.356358"
      y2="25.822845"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="45.865936"
      x2="68.869011"
      y1="26.273958"
      y2="17.80747"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="0.1794975"
      x2="22.731525"
      y1="66.819733"
      y2="58.35323"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="23.087475"
      x2="46.090553"
      y1="58.352058"
      y2="66.818542"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="45.734657"
      x2="68.286682"
      y1="66.819733"
      y2="58.35323"
    />
    <RNSVGCircle
      cx="34.412552"
      cy="14.1218"
      fill={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      r="14.1218"
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
    />
    <RNSVGPath
      d="m 34.116455,42.4782 -13.6445,-24.5226 27.7313,0.2453 z"
      fill={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`renders in dark mode correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="#ffffff"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  stroke="#d9d9d9"
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={67.297112}
  vbWidth={69.470993}
  width={24}
>
  <RNSVGGroup
    fill={
      {
        "payload": 4294967295,
        "type": 0,
      }
    }
    propList={
      [
        "fill",
        "stroke",
      ]
    }
    stroke={
      {
        "payload": 4292467161,
        "type": 0,
      }
    }
  >
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      x1="1.3100156"
      x2="0.85401821"
      y1="26.296127"
      y2="67.291687"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      x1="68.971024"
      x2="68.51503"
      y1="17.829628"
      y2="58.825188"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      x1="46.419025"
      x2="45.963028"
      y1="26.296127"
      y2="67.291687"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      x1="23.866926"
      x2="23.410929"
      y1="17.829628"
      y2="58.825188"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="0.63054752"
      x2="23.182575"
      y1="25.82403"
      y2="17.357529"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="23.087475"
      x2="46.090553"
      y1="17.356358"
      y2="25.822845"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="45.865936"
      x2="68.869011"
      y1="26.273958"
      y2="17.80747"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="0.1794975"
      x2="22.731525"
      y1="66.819733"
      y2="58.35323"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="23.087475"
      x2="46.090553"
      y1="58.352058"
      y2="66.818542"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="45.734657"
      x2="68.286682"
      y1="66.819733"
      y2="58.35323"
    />
    <RNSVGCircle
      cx="34.412552"
      cy="14.1218"
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      r="14.1218"
      stroke={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
    />
    <RNSVGPath
      d="m 34.116455,42.4782 -13.6445,-24.5226 27.7313,0.2453 z"
      fill={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4294967295,
          "type": 0,
        }
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;

exports[`renders uncolored correctly 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="#d9d9d9"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  stroke="#000000"
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={67.297112}
  vbWidth={69.470993}
  width={24}
>
  <RNSVGGroup
    fill={
      {
        "payload": 4292467161,
        "type": 0,
      }
    }
    propList={
      [
        "fill",
        "stroke",
      ]
    }
    stroke={
      {
        "payload": 4278190080,
        "type": 0,
      }
    }
  >
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="1.3100156"
      x2="0.85401821"
      y1="26.296127"
      y2="67.291687"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="68.971024"
      x2="68.51503"
      y1="17.829628"
      y2="58.825188"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="46.419025"
      x2="45.963028"
      y1="26.296127"
      y2="67.291687"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      x1="23.866926"
      x2="23.410929"
      y1="17.829628"
      y2="58.825188"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="0.63054752"
      x2="23.182575"
      y1="25.82403"
      y2="17.357529"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="23.087475"
      x2="46.090553"
      y1="17.356358"
      y2="25.822845"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="45.865936"
      x2="68.869011"
      y1="26.273958"
      y2="17.80747"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="0.1794975"
      x2="22.731525"
      y1="66.819733"
      y2="58.35323"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="23.087475"
      x2="46.090553"
      y1="58.352058"
      y2="66.818542"
    />
    <RNSVGLine
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      propList={
        [
          "stroke",
          "strokeWidth",
        ]
      }
      stroke={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      strokeWidth="0.999984"
      x1="45.734657"
      x2="68.286682"
      y1="66.819733"
      y2="58.35323"
    />
    <RNSVGCircle
      cx="34.412552"
      cy="14.1218"
      fill={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      r="14.1218"
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
    />
    <RNSVGPath
      d="m 34.116455,42.4782 -13.6445,-24.5226 27.7313,0.2453 z"
      fill={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
          "stroke",
        ]
      }
      stroke={
        {
          "payload": 4292467161,
          "type": 0,
        }
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
